members-only/app/views/posts/index.html.erb

```diff
@@ -0,0 +1,21 @@
+
+	<div class="row py-5">
+		
+
+   <div class="col-12">
+   	<h1 class="px-3">Secret Posts</h1>
+
+   <% if logged_in? %>
+    <p class="px-3 d-block"><%= @user.name %> is currently logged in</p>
```

Comment:
I don't really understand why you need this.
What if you directly use `current_user`?

Reply:
We did this as well